nbodykit/base/catalog.py

```diff
@@ -65,70 +77,72 @@ def __str__(self):
             r = r + " last: %s" % str(self[-1].compute())
         return r
 
-def column(name=None):
+def column(name=None, default=False):
     """
-    Decorator that defines a function as a column in a CatalogSource
+    Decorator that defines the decorated function as a column in a
+    CatalogSource.
+
+    This can be used as a decorator with or without arguments. If no ``name``
+    is specified, the function name is used.
+
+    Parameters
+    ----------
+    name : str, optional
+        the name of the column; if not provided, the name of the function
+        being decorated is used
+    default : bool, optional
```

Comment:
I wonder if the name 'fallback' is better suited than default in this context.

Reply:
I went with "is_default" here and below to unify the syntax. I think we are now using "is_default" everywhere so the syntax reads "column.is_default"